### DISCUSSION TITLE
Should iroh start have the ability to add data?

### DISCUSSION THREAD

**Discussion by rklaehn** [n0-computer/iroh#1659] [Ideas]
iroh start starts a daemon. People usually don't expect to be able to provide data here.

Instead, we should have a separate command that just starts a temp node and serves some data, like the old `sendme provide`. This should share most code so testing is still meaningful, and be used in netsim.

**Comment by rklaehn** [comment]
https://discord.com/channels/949724860232392765/950683937661935667/1154360922438119444

**Comment by b5** [comment]
An alternative we've discussed: _keep_ the ability to add data on start, and give `iroh start` a new `--temp` command that has it spin up the entire node in-memory.

The prior art for all of this is [`sendme provide`](https://github.com/n0-computer/sendme/blob/3a3fc46ba9faef7deff9f63a9b36e9a7ed00655e/src/main.rs#L28-L45), which was a very nice, fast way to just say "hey, serve this until the process stops"
